mkosi/config.py

```diff
@@ -56,15 +56,20 @@ class Verb(StrEnum):
     help          = enum.auto()
     genkey        = enum.auto()
     documentation = enum.auto()
+    journalctl    = enum.auto()
+    coredumpctl   = enum.auto()
 
     def supports_cmdline(self) -> bool:
-        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.ssh)
+        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.ssh,
+                        Verb.journalctl, Verb.coredumpctl)
 
     def needs_build(self) -> bool:
-        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.serve)
+        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu,
+                        Verb.serve, Verb.journalctl, Verb.coredumpctl)
 
     def needs_root(self) -> bool:
-        return self in (Verb.shell, Verb.boot)
+        return self in (Verb.shell, Verb.boot, Verb.journalctl,
+                        Verb.coredumpctl)
```

Comment:
No need to break the lines here, all of them are short enough to fit on a single line within our current line limit.

Reply:
Okay I will fix these and squash it, this was just where my editor broke it and I assumed it had found some config file somewhere to tell it to do that 🤷

Reply:
This is fixed now and the commits have been squashed.